### ISSUE TITLE
Frameworks/nutble.framework: code object is not signed at all

### ISSUE THREAD

**Issue by HashirHexeR8** [nut-tech/nut-sdk-ios#3]
as the title describes i am having this issue while trying to upload my app to test flight, the app upload fails during the signing process with following error: 
<img width="688" alt="image" src="https://user-images.githubusercontent.com/60361519/204948897-65d31571-43e2-470f-9d25-6c5c1c03b1e4.png">

**Comment by bingerz** [comment]
Is the version of nutale.framework referenced by your app in the release directory?
[release/nutble.framework](https://github.com/nut-tech/nut-sdk-ios/tree/master/framwork/release/nutble.framework)

**Comment by HashirHexeR8** [comment]
I'll confirm right away

**Comment by HashirHexeR8** [comment]
<img width="798" alt="image" src="https://user-images.githubusercontent.com/60361519/205432764-39c262f5-6f3d-4d07-8a61-bffcc08618bb.png">

Here's the complete details of the framework that i'm using in my app.

**Comment by bingerz** [comment]
I will reproduce your situation and get back to you, please give me some time.

**Comment by bingerz** [comment]
I will reproduce your situation and get back to you, please give me some time.

**Comment by HashirHexeR8** [comment]
Sure please let me know if you need anything from my side.

**Comment by HashirHexeR8** [comment]
Hey @bingerz were you able to reproduce the issue?

**Comment by nutspace** [comment]
I didn't reproduce the same error, but I found an Unsupported Architecture issue, and I can remove the unsupported architecture with the following command, I can upload it successfully now, looking forward to your try and feedback.

lipo -remove i386 nutble.framework/nutble -o nutble.framework/nutble
lipo -remove x86_64 nutble.framework/nutble -o nutble.framework/nutble

**Comment by HashirHexeR8** [comment]
Yes that fixed my issue thanks for the update, could you please update the repo with this framework for future usages?

**Comment by nutspace** [comment]
I will update this nutble.framework, please use the method of ”lipo remove“ first.
